pkgs/applications/audio/sonobus/default.nix

```diff
@@ -19,14 +19,14 @@
 , webkitgtk
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "sonobus";
-  version = "1.7.0";
+  version = "1.7.2";
 
   src = fetchFromGitHub {
     owner = "sonosaurus";
     repo = "sonobus";
-    rev = version;
+    rev = finalAttrs.version;
     sha256 = "sha256-zOPQK5X1E6t53DOjV7qSelyep4+m9aL4tRHqwyeuFQA=";
```

Comment:
why is sha256 not updated ? also it should be using `hash` attribute

Reply:
oops! my bad

Reply:
weird, on my machine it does say finalAttrs. Not sure what happened there.